plugins/TagRemove_Layer.py

```diff
@@ -47,7 +47,7 @@ def way(self, data, tags, nds):
             elif tags.get(u"highway") and tags.get(u"highway") != "steps" and (not tags.get(u"indoor") or tags.get(u"indoor") == "no"):
                 if layer[0] == "-" and (not tags.get(u"tunnel") or tags.get(u"tunnel") == "no"):
                     return {"class": 41104 if len(nds) > 3 else 41103, "subclass": 0}
-                elif layer[0] != "-" and (not tags.get(u"bridge") or tags.get(u"bridge") == "no"):
+                elif layer[0] != "-" and (not tags.get(u"bridge") or tags.get(u"bridge") == "no" or tags.get(u"man_made") == "pier"):
```

Comment:
I think it should be `!= 'pier'`. But to be more clear
```suggestion
                elif layer[0] != "-" and (not tags.get(u"bridge") or tags.get(u"bridge") == "no") and tags.get(u"man_made") != "pier":
```